tests.py

```diff
@@ -81,5 +104,33 @@ def test_subclauses_arithmetic_booleans(self):
         self.run_vsm('subclauses_arithmetic_booleans', input_data,
                 expected_output, False)
 
+
+class TestIPC(TestVSM):
+
+    def setUp(self):
+        load_plugin('ipc.{}'.format(IPC_MODULE))
+
+    def test_simple0(self):
+        input_data = [('transmission_gear', '"reverse"')]
```

Comment:
You can build the input data structure by processing the string. Like, you can split at =, strip spaces.